release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -2391,11 +2391,125 @@ An update for `jenkins` and `openshift` is now available for {product-title}
+...
+----
+
+For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1868751[*BZ#1868751*].
+
+[id="ocp-4-5-8-bug-fixes"]
+==== Bug fixes
+
+* Previously, intermittent API server errors were reported for the
+`ImageChangesInProgress` condition instead of the `SamplesExists` condition of
+the Samples Operator config object. When the API server reported that all
+samples were installed, the Samples Operator failed to switch the `Progressing`
+condition to `false` because there was unexpected data in its
+`ImageChangesInProgress` condition. This caused upgrades to incorrectly be
+marked as incomplete. This bug fix updates the `SamplesExists` condition to
+report errors on the API server, so upgrades are no longer blocked if
+intermittent API server errors occur while the Samples Operator is upgrading.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1857201[BZ#1857201])
+
+* Previously, the `ironic-image` container configuration was missing the setting
+to enable the `idrac-redfish-virtual-media` boot driver. Because of this, users
+were unable to select the `idrac-virtual-media` boot URL for Metal3. The missing
+`ironic-image` container configuration is now included, so users are able to
+select the `idrac-virtual-media` URL for Metal3.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1859488[BZ#1859488])
+
+* Previously, the Operand form array and object fields did not have logic to
+retrieve and show field descriptions on the form. In result, no descriptions
+were rendered for array or object type fields. This bug fix adds logic to now
+display array and object field descriptions on the Operand creation form.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1861433[BZ#1861433])
+
+* Previously, Buildah erased image architecture and OS fields on images. This
+caused common container tools to fail because the resulting images could not
+identify their architecture and OS. This bug fix forces Buildah to not overwrite
+the image and architecture unless there are explicit overrides, which ensures
+images always have architecture and OS fields, and the image mismatch warning
+does not appear.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1868401[BZ#1868401])
+
+* Previously, intermittent invalid memory address or nil pointer dereference
+errors occurred followed by timeouts for Kube API access when running CoreDNS
+1.6.6. This is now fixed by correctly handling errors with Endpoint Tombstones.
+Now CoreDNS behaves as intended without intermittent panics.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1869309[BZ#1869309])
+
+* Previously, the controller for BareMetalHost objects mirrored status data to
+an annotation, including a timestamp of the latest status update. This was not
+needed by the cluster. This could result in the BareMetalHost entering a state
+of continuous flux where affected BareMetalHosts would be subject to longer
+back-offs between reconciliation to prevent the controller overwhelming the
```

Comment:
s/to prevent the controller overwhelming/to prevent the controller from overwhelming/